README.md

```diff
@@ -19,7 +19,10 @@ Installing all dependencies
 
 - Navigate to `localhost:3456` in Google Chrome browser
 
-## Test
+## Setup Testframework
+- Step by step instructions to setup JEST is found at https://github.com/DynamoDS/librarie.js/wiki/Setting-up-test-frameworks-for-librarie.js
+
+## Run Tests
```

Comment:
Previous comments in [another PR](https://github.com/DynamoDS/librarie.js/pull/62) are not addressed.